src/wmnds/components/form-elements/radios/_properties.njk.json

```diff
@@ -0,0 +1,115 @@
+      "arrayOptions": [
+        {
+          "name": "id",
+          "type": "string",
+          "description": "Optional id attribute to add to the hint span tag."
+        },
+        {
+          "name": "contentText",
+          "type": "string",
+          "description": "<strong>Required.</strong> If <code class='wmnds-website-inline-code'>contentHTML</code> is set, this is not required. Text to use within the hint. If <code class='wmnds-website-inline-code'>contentHTML</code> is provided, the <code class='wmnds-website-inline-code'>contentText</code> argument will be ignored."
+        },
+        {
+          "name": "contentHTML",
+          "type": "string",
+          "description": "<strong>Required.</strong> If <code class='wmnds-website-inline-code'>contentText</code> is set, this is not required. HTML to use within the hint. If <code class='wmnds-website-inline-code'>contentHTML</code> is provided, the <code class='wmnds-website-inline-code'>contentText</code> argument will be ignored."
+        },
+        {
+          "name": "classes",
+          "type": "string",
+          "description": "Classes to add to the hint span tag."
+        }
+      ]
+    },
+    {
+      "name": "idPrefix",
+      "type": "string",
+      "description": "String to prefix id for each radio item if no id is specified on each item. If not passed, fall back to using the name option instead."
+    },
+    {
+      "name": "inline",
+      "type": "boolean",
+      "description": "Determines whether to the display the radios horizontally or vertically."
+    },
+    {
+      "name": "name",
+      "type": "string",
+      "description": "<strong>Required.</strong> Name attribute for all radio items."
+    },
+    {
+      "name": "classes",
+      "type": "string",
+      "description": "Classes to add to the radios container."
+    },
+    {
+      "name": "error",
+      "type": "boolean",
+      "description": "If true, errorMessage will show. Defaults to <code class='wmnds-website-inline-code'>false</code>."
+    },
+    {
+      "name": "errorMessage",
+      "type": "object",
+      "description": "Options for the error message component. See <code class='wmnds-website-inline-code'>errorMessage</code> options below.",
+      "arrayOptions": [
+        {
+          "name": "id",
+          "type": "string",
+          "description": "Id attribute to add to the error message span tag."
+        },
+        {
+          "name": "contentText",
+          "type": "string",
+          "description": "<strong>Required.</strong> Text to use within the error message. If <code class='wmnds-website-inline-code'>contentHTML</code> is supplied, this is ignored."
+        },
+        {
+          "name": "contentHTML",
+          "type": "string",
+          "description": "<strong>Required.</strong> HTML to use within the error message"
+        }
+      ]
```

Comment:
I've a classes param here too. Do you agree adding it or do you prefer removing it from the other places? I don't believe it will be a need.. but label has it and I guess I follow it...

Reply:
@catiarodriguescosta yeah, I was also thinking this when I saw that you added it but I don't think it will hurt to leave it in there... although I doubt it will get used. @daylesalmon what do you think?

Reply:
Can you add it just to be consistent with the other, please? :)

Reply:
Sorry I misunderstood your message, I didn't realise the you were saying the prop was missing

Reply:
No problem! :)